code/grant/html_lab05/static/styles.css

```diff
@@ -0,0 +1,100 @@
+
+
+
+
+
+
```

Comment:
Moving forward you should be able to completely move away from the use of raw CSS. You still want to remember it for when you need it, but you shouldn't need it super often.